token/services/db/sql/common/identity.go

```diff
@@ -86,8 +85,8 @@ func NewCachedIdentityStore(
 		readDB,
 		writeDB,
 		tables,
-		secondcache.NewTyped[bool](5000),
-		secondcache.NewTyped[[]byte](5000),
+		cache2.NewNoCache[bool](),
```

Comment:
Just reading this line - I am a bit clueless what is instantiated here a new no cache from the cache2 package?

Reply:
As this is the constructor `NewCachedIdentityStore`, shouldn't it use the ristretto-based cache impl?